packages/block-library/src/table/style.scss

```
@@ -11,6 +11,11 @@
 		width: 100%;
 	}
 
+	tfoot {
+		font-weight: 700;
+    	text-align: inherit;
```

It looks like this line needs indenting to align with the previous line:

```suggestion
    		text-align: inherit;
```